### PR TITLE
Feature python code

### PR DESCRIPTION
**Pull request by shirishma1**
Now it is able to read both English and Irish files and spellcheck accordingly

### CODE REVIEW

File: spellcheck.py [hunk 1]
```diff
@@ -63,30 +83,42 @@ def check():
...
+        
+            print('in irish')
+        elif language == 'english':
+            unique_words = unique_words_english
+            word_probability = word_probability_english
+
         for word in iwords:
             guesses = correct_spelling(word, unique_words, word_probability)
-            toporder = sorted(guesses, key=lambda x: x[1], reverse=True)[:len(guesses)]  # arranging suggestions in decreasing order
+            print('after guesses')
```
Comment (Dheeraj-Thikkamaneni) [inline spellcheck.py]: Is this code for your reference to check where it is in your terminal ?

**Review by Dheeraj-Thikkamaneni — COMMENTED** [review]
looks all good

File: spellcheck.py (hunk 1 above)
Comment (shirishma1) [reply spellcheck.py]: I can delete it if you'd like

Comment (shirishma1) [reply spellcheck.py]: I deleted it.

**Pull request merged**